### DISCUSSION TITLE
Update to 0.9.7 using CLI

### DISCUSSION THREAD

**Discussion by BumpaRoy** [Azure/data-api-builder#1913] [Q&A]
I am currently using v0.8.49 of dab.  Using the CLI, I tried the command: "dotnet tool update -g Microsoft.DatApiBuilder --version 0.9.7" and it did not work.  Telling me: 

No packages exist with this id in source(s): C:\Program Files\dotnet\library-packs,  ComponentOne Local, Microsoft Visual Studio Offline Packages, nuget.org, Telerik NuGet Source Tool 'microsoft.datapibuilder' failed to update due to the following: The tool package could not be restored.

Can anybody point me in the right direction?

**Comment by severussundar** [comment]
Hi @BumpaRoy , could you please try updating with the following command  

`dotnet tool update -g --add-source 'https://api.nuget.org/v3/index.json' Microsoft.DataApiBuilder --version 0.9.7 `
